woocommerce/woocommerce-gutenberg-products-block
# Migrate cart/checkout tax tests to Playwright

## What

Moves the tax tests from puppeteer to Playwright

Fixes #11896

## Why

To move away from Puppeteer.


## Testing Instructions



_Please consider any edge cases this change may have, and also other areas of the product this may impact._

1. Run tests, ensure CI passes.


## Screenshots or screencast



| Before | After |
| ------ | ----- |
|        |       |

## WooCommerce Visibility


Required:


## Checklist

Required:


Conditional:


## Changelog


> Add suggested changelog entry here.
